test/python/test_pack.py

```diff
@@ -263,7 +263,18 @@ def test_packing_without_git(tmpdir):
         "--version", "0.1.0",
         project_path,
     ]
-    process = subprocess.run(cmd, cwd=tmpdir)
+    subprocess.run(cmd, cwd=tmpdir)
+    assert 'test-project-0.1.0-0.rpm' in os.listdir(tmpdir)
+
+    # pass extended version
```

Comment:
Why didn't you add a new test for this feature?

BTW, maybe we should add `project_without_dependencies` or smth like this to use it in this kind of tests.

Reply:
See explanation here https://github.com/tarantool/cartridge-cli/pull/114/files#r367650797